src/components/browse/PrimaryCallToAction.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="primary-cta">
+    <RichText
+      :text="text"
+      :rich-text-is-card="false"
+      class="primary-cta-rich-text"
+    />
+    <SmartLink
+      :destination="link.url"
+      data-qa="call to action"
+      class="btn btn-cta btn-primary d-inline-flex align-items-center"
```

Comment:
The classes `d-inline-flex align-items-center` doesn't seem to make a difference. When I remove them, the styling looks the same.